nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/system-diagnostics-dialog/system-diagnostics-dialog.component.html

```diff
@@ -19,12 +19,12 @@
     <h2 mat-dialog-title>System Diagnostics</h2>
     <error-banner></error-banner>
     <div class="system-diagnostics">
-        <mat-dialog-content>
-            <div class="dialog-content">
-                <mat-tab-group>
-                    <mat-tab label="JVM">
-                        <div class="tab-content py-4 h-full w-full">
-                            <div class="inset-0 flex gap-y-4">
+        <div class="dialog-content">
```

Comment:
this `dialog-content` class is not needed and makes the dialog have double scrollbars.
<img width="798" alt="Screenshot 2024-05-09 at 9 40 19 AM" src="https://github.com/apache/nifi/assets/713866/a4143d0d-bb47-4b11-b7d3-c6e241e2de5d">